### PR TITLE
PAR: Add news modal

### PR DESCRIPTION
**Pull request by NappiLe**
Screenshot 2022-05-30 160210

### CODE REVIEW

**Comment by Rostislav013** [comment]
Should it be centered vertically too or is it fine like this too?
image

**Comment by NappiLe** [comment]
It is vertically centered now

**Review by Rostislav013 — APPROVED** [review]
lgtm, but change screenshots and change PR name

File: src/Nordea/Components/Modal.elm [hunk 1]
```diff
@@ -50,14 +55,45 @@ import Nordea.Resources.Icons as Icons
...
+type Variant
+    = DefaultModal
+    | NewsModal
+
+
+type alias Config msg =
+    { variant : Variant
+    , onClickClose : msg
     , title : String
+    , subtitle : String
```
Comment (fredrikloberg) [inline src/Nordea/Components/Modal.elm:67]:
```suggestion
    , title : Maybe String
    , subtitle : Maybe String
```

**Review by fredrikloberg — APPROVED** [review]
Looks good!

**Pull request merged**